include/seqan3/search/views/kmer_hash.hpp

```diff
@@ -774,6 +774,9 @@ namespace seqan3::views
  *
  * See the \link views views submodule documentation \endlink for detailed descriptions of the view properties.
  *
+ * \attention The Shape is defined from right to left. The mask 0b1101 applied to ACGT will return
+ * the sequence AGT.
```

Comment:
why is this an `attention`? Isn't this intuitive to read the shape from left to right?

Also:
```suggestion
 * \note The Shape is defined from right to left. The mask 0b1101 applied to "ACGT" is 
 * interpreted as "A.GT" will return the hash value for "AGT".
```